sequencer/sequencer.go

```diff
@@ -213,6 +213,13 @@ func (s *Sequencer) purgeOldPoolTxs(ctx context.Context) {
 			continue
 		}
 		log.Infof("deleted %d selected txs from the pool", len(txHashes))
+
+		// Delete failed txs older than a certain date (14 seconds per L1 block)
+		err = s.pool.DeleteFailedTransactionsOlderThan(ctx, time.Now().Add(-time.Duration(s.cfg.BlocksAmountForTxsToBeDeleted*14)*time.Second)) //nolint:gomnd
+		if err != nil {
+			log.Errorf("failed to delete txs from the pool, err: %v", err)
```

Comment:
"failed to delete failed txs from the pool" (as now is the same message as we have in line 212, in this way we can differentiate both)